dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -227,13 +227,13 @@ FEATURE_NOT_IMPLEMENTED.useraction=Avoid using the feature, or switch to a drive
 # Using 4000-4999 for Liberty-only messages
 # 
 
-DSRA4000.no.suitable.driver=DSRA4000E: A valid JDBC driver implementation class was not found for the jdbcDriver {0} using the library {1}.
-DSRA4000.no.suitable.driver.explanation=A valid JDBC driver implementation class was not found.
-DSRA4000.no.suitable.driver.useraction=Update the configuration so that either the library contains a path to a recognized JDBC driver or the implementation class is specified on the data source.
+DSRA4000.no.suitable.driver=DSRA4000E: No implementations of {0} are found for {1} with library {2}. Searched in: {3}. Searched under packages: {4}.
+DSRA4000.no.suitable.driver.explanation=Reasons why a data source or driver implementation might not be found include: Name or location of JDBC driver JAR files might be incorrect or inaccessible. JDBC driver might not properly identify its Driver class, or might be lacking one. Liberty's built-in logic to identify data source implementation classes does not apply to all drivers.
+DSRA4000.no.suitable.driver.useraction=Verify that configuration of the JDBC driver and library are correct. If implementation class are still not found, manually specify them in the configuration.
```

Comment:
This message appears in both sets, and has a little singular/ plural ambiguity. Is the "JDBC driver and library" a single configuration? Or is it "JDBC driver and JDBC library" (two separate configurations). 

DSRA4000.no.suitable.driver.useraction=Verify that **the configurations** of the JDBC driver and library are correct. If **the implementation classes** are still not found, manually specify them in the configuration.

Reply:
Currently jdbcDriver and library are separate configuration elements that must be configured. However, there is a feature planned in the backlog that makes jdbcDriver configuration optional.  In anticipation of this feature, I was intentionally ambiguous about whether there is a single or multiple configuration element.  I'm fine to change it as you suggested.  We can always revisit this useraction when that other feature eventually goes in.